src/network/webtask.cpp

```diff
@@ -173,19 +173,20 @@ void WebTask::slotStart(int timeoutMillis) {
                 QByteArray());
         return;
     }
+    DEBUG_ASSERT_QOBJECT_THREAD_AFFINITY(pNetworkAccessManager);
 
     kLogger.debug()
             << this
             << "Starting...";
 
-    m_pendingNetworkReply = doStartNetworkRequest(
-            m_networkAccessManager,
+    auto* const pPendingNetworkReply = doStartNetworkRequest(
```

Comment:
```suggestion
    QNetworkReply* const pPendingNetworkReply = doStartNetworkRequest(
```
The type is out of sight.

Reply:
The actual type is irrelevant here and the temporary variable isn't even necessary. I could assign it directly to the weak pointer and will do so instead of adding superfluous, redundant type noise.

PS: This auto-nitpicking is annoying.